Rules/CTF/Scripts/CTF_Trading.as

```diff
@@ -100,7 +101,7 @@ void onPlayerDie(CRules@ this, CPlayer@ victim, CPlayer@ killer, u8 customData)
 		{
 			if (killer !is victim && killer.getTeamNum() != victim.getTeamNum())
 			{
-				killer.server_setCoins(killer.getCoins() + coinsOnKillAdd);
+				killer.server_setCoins(killer.getCoins() + coinsOnKillAdd * (1+killer.get_u8("killstreak")*killstreakFactor));
```

Comment:
This equation is not correct. When killstreakFactor is 1, I would expect there to be no bonus for killstreaks. And if killstreakFactor is 2, I would expect the coin bonus to double with each kill.

Reply:
My mistake, I misinterpreted the equation.